python/fledge/plugins/south/modbustcp/modbustcp.py

```diff
@@ -211,29 +211,41 @@ def plugin_poll(handle):
         coils_address_info = modbus_map['coils']
         if len(coils_address_info) > 0:
             for k, address in coils_address_info.items():
-                coil_bit_values = mbus_client.read_coils(99 + int(address), 1, unit=unit_id)
-                readings.update({k: coil_bit_values.bits[0]})
+                coil_bit_values = mbus_client.read_coils(int(address), 1, unit=unit_id)
+                if coil_blit_values is None:

```

Comment:
coil_blit_values unknown variable! should be coil_bit_values

Reply:
Thanks for picking that up